cryptol-remote-api/python/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Revision history for `cryptol` Python package
 
+## next
+
+* Add `modules` command to return the modules, submodules and associated
+  documentation for them.
```

Comment:
Can you add a similar changelog entry to the [`cryptol-remote-api` changelog](https://github.com/GaloisInc/cryptol/blob/670262219d9e1b001c2130cc81c3ce3680cd8029/cryptol-remote-api/CHANGELOG.md)?